modules/c++/math.linear/include/math/linear/Matrix2D.h

```diff
@@ -1082,6 +1082,21 @@ class Matrix2D
     }
 
     /*!
+     *  Negation operator;
+     *
+     *  \code
+           B = -A;
+     *  \endcode
+     *
+     */
+    Matrix2D operator-() const
+    {
+        Matrix2D mneg(*this);
+        std::transform(mneg.mRaw, mneg.mRaw+mneg.mMN, mneg.mRaw, std::negate<_T>());
```

Comment:
Hm, compiler didn't complain (gcc or VS2015). I'll add it in case it's getting pulled in somewhere else.